src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+export const addBook = () => ({
+  type: 'ADD_BOOK',
+});
+
+export const removeBook = () => ({
+  type: 'REMOVE_BOOK',
+});
```

Comment:
- [ ] Kindly define the action types `ADD_BOOK` and `REMOVE_BOOK` as variables and it should follow the duck pattern and then use it in your types as requested in the project requirements. for instance
``` javascript
  const ADD_BOOK = books/books/ADD_BOOK
```